src/styles/content/tables/styles.less

```diff
@@ -12,13 +12,6 @@
 
   .table {
 
-    &.table-header-nowrap {
-
-      th {
-        white-space: nowrap;
```

Comment:
I bet this breaks something somewhere else… 😞 
Also, it is about `th`, is it necessary for this PR? 🤔

Reply:
@juliangieseke it is redundant because it is nowrap by default on th, so I decided to remove it. I made sure to check all instances were the class was used and I removed it there as well.

Not exactly necessary, but I figured I would remove some unnecessary code 🤷‍♂️

Reply:
Ah ok, I think it is a good idea then to keep bugfix commits separate from refactor commits. having the bugfix only fixing the bug makes it easier to cherry-pick and blame later on if necessary 🙇